deploy/ansible/playbooks/iroha-docker-cluster/iroha-deploy.yml

```diff
@@ -0,0 +1,16 @@
+  - hosts: locals
```

Comment:
should contain comments, specifying:
* why do we need python2 as a pre-task
* what are the roles
* why do we need tags: ["deliver", "deploy"] in iroha-cluster-deploy-node